app/serializers/ticket_blueprint.rb

```diff
@@ -4,4 +4,13 @@ class TicketBlueprint < Blueprinter::Base
   field :worker_name do |ticket,options|
     " #{ticket.worker.first_name} #{ticket.worker.last_name}"
   end
+  field :comment_count do |ticket,options|
+    amount = 0
+    Comment.all.each do |comment|
```

Comment:
okay, looks like you need to get more familiar with ActiveRecord queries 

in this case it should be just like `Comment.where(ticket_id: ticket.id).count` or even `ticket.comments.count`